src/coreclr/jit/emitxarch.cpp

```diff
@@ -16261,6 +16276,15 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             break;
         }
 
+        case INS_shlx:
+        case INS_sarx:
+        case INS_shrx:
+        {
+            result.insLatency    = PERFSCORE_LATENCY_2C;
```

Comment:
```suggestion
            result.insLatency    += PERFSCORE_LATENCY_1C;
```

It should be similar to that of `rorx` AFAIK.

![image](https://user-images.githubusercontent.com/12488060/160756512-b67f3acf-e807-422d-b721-ec24c7e4e12f.png)

Reply:
Qhere are you getting that number from? I think that matches the newer hardware and not Skylake, which is what we have used for the other numbers

Reply:
Updated to `result.insLatency    += PERFSCORE_LATENCY_1C;`.